src/components/Miscellaneous/Tabs/Tabs.vue

```diff
@@ -12,7 +12,18 @@
         class="tab-content"
         :style="verticalTabs ? `margin-bottom: ${selectedTab === tab ? 9 : 21}px !important;` : ''"
       >
+        <div
+          v-if="selectedTab !== tab"
+          class="icon"
+        >
+          <PbIcon
+            v-if="hasIcon"
+            :icon="`${icon} fa-xs`"
```

Comment:
Você esta adicionando o mesmo icone para todas as tabs!!!

Reply:
não to mais não :)))